starter-code/server.js

```diff
@@ -121,7 +122,7 @@ app.delete('/articles/:id', function(request, response) {
 
 app.delete('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Images two and five. Article.prototype.deleteRecord. Delete.
```

Comment:
Image 3, same as comment above
This function is deleting all the articles in the table, so the method is `Article.truncateTable`